client/test/router_store.spec.js

```diff
@@ -0,0 +1,669 @@
+        assert.equal(routerStore.getSelectedTab(), 'account');
+        assert.equal(currentURL, `#mailbox/${fixtures.account.inboxMailbox}`);
+        done();
+      }, 5000);
+    }); // .timeout(6000);
+    it('ADD_ACCOUNT_FAILURE', () => {
+      dispatcher.dispatch({
+        type: ActionTypes.ADD_ACCOUNT_FAILURE,
+        value: { error: fixtures.testError },
+      });
+      assert.isFalse(routerStore.isWaiting());
+      let errors = routerStore.getErrors().toObject();
+      assert.deepEqual(errors, fixtures.unknownError);
+      dispatcher.dispatch({
+        type: ActionTypes.ADD_ACCOUNT_FAILURE,
+        value: { error: fixtures.fullTestError },
+      });
+      assert.isFalse(routerStore.isWaiting());
+      errors = routerStore.getErrors().toObject();
+      assert.deepEqual(errors, {
+        field1: {
+          message: fixtures.fullTestError.field,
+          originalError: fixtures.fullTestError.originalError,
+          originalErrorStack: fixtures.fullTestError.originalErrorStack,
+        },
+        field2: {
+          message: fixtures.fullTestError.field,
+          originalError: fixtures.fullTestError.originalError,
+          originalErrorStack: fixtures.fullTestError.originalErrorStack,
+        },
+      });
+    });
+    it('CHECK_ACCOUNT_REQUEST', () => {
+      dispatcher.dispatch({ type: ActionTypes.CHECK_ACCOUNT_REQUEST });
+      assert.isTrue(routerStore.isChecking());
+    });
+    it('CHECK_ACCOUNT_SUCCESS', () => {
+      dispatcher.dispatch({ type: ActionTypes.CHECK_ACCOUNT_SUCCESS });
+      assert.isFalse(routerStore.isChecking());
+    });
+    it('CHECK_ACCOUNT_FAILURE', () => {
+      dispatcher.dispatch({
+        type: ActionTypes.CHECK_ACCOUNT_FAILURE,
+        value: { error: fixtures.testError },
+      });
+      assert.isFalse(routerStore.isChecking());
+      let errors = routerStore.getErrors().toObject();
+      assert.deepEqual(errors, fixtures.unknownError);
+      dispatcher.dispatch({
+        type: ActionTypes.CHECK_ACCOUNT_FAILURE,
+        value: { error: fixtures.fullTestError },
+      });
+      assert.isFalse(routerStore.isChecking());
+      errors = routerStore.getErrors().toObject();
+      assert.deepEqual(errors, {
+        field1: {
+          message: fixtures.fullTestError.field,
+          originalError: fixtures.fullTestError.originalError,
+          originalErrorStack: fixtures.fullTestError.originalErrorStack,
+        },
+        field2: {
+          message: fixtures.fullTestError.field,
+          originalError: fixtures.fullTestError.originalError,
+          originalErrorStack: fixtures.fullTestError.originalErrorStack,
+        },
+      });
+    });
+    it('EDIT_ACCOUNT_REQUEST', () => {
+      dispatcher.dispatch({
+        type: ActionTypes.EDIT_ACCOUNT_REQUEST,
+        value: { rawAccount: _.clone(fixtures.account) },
+      });
+      assert.isTrue(routerStore.isWaiting());
+    });
+    it('EDIT_ACCOUNT_SUCCESS', () => {
```

Comment:
You should update your branch; this has moved from `routerStore` to `requestsStore`.